test/native_objc_test/setup.dart

```diff
@@ -27,6 +27,27 @@ Future<void> _buildLib(String input, String output) async {
   print('Generated file: $output');
 }
 
+Future<void> _buildSwift(
+    String input, String outputHeader, String outputLib) async {
+  final args = [
+    '-c',
+    input,
+    '-emit-objc-header-path',
+    outputHeader,
+    '-emit-library',
+    '-o',
+    outputLib,
+  ];
+  final process = await Process.start('swiftc', args);
+  unawaited(stdout.addStream(process.stdout));
```

Comment:
I don't understand. This script is not invoked by `dart test`. The `await process.exitCode` line should be enough to get all the output to print. Did you have another suggestion?

Reply:
i've had issues where `process.exitCode` completed and I wrote something with `exit(nonZero)` synchronously after that, and that meant the microTaskQueue wasn't drained and the pending `process.stderr` stream events weren't processed, leading to truncated output.

I worked around that by having an `onDone` that completed a completer and then awaiting the exitCode and completer.future for both the stdout and stderr.

Anyway, it might be a bit heavyweight for what we're doing here.

> `dart test`

My bad, you're right :)